apidocs/conf.py

```diff
@@ -0,0 +1,51 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+import sys
+
+# Add custom paths for modules
+sys.path.insert(0, os.path.abspath("../plugins"))
+sys.path.insert(1, os.path.abspath("../scripts"))
+sys.path.insert(2, os.path.abspath("../Surfactant"))
```

Comment:
Instead of doing this, add an additional block in this file to "install" Surfactant from source: https://github.com/LLNL/Surfactant/blob/7f2498e2dccf23bd7a5554bd923007d006daaf0b/.readthedocs.yaml#L31

Should be able to do that by adding another entry under "install" that looks something like this:

```yaml
  - method: pip
    path: .
    extra_requirements:
    - docs
```